src/rrules/blas.jl

```diff
@@ -36,6 +38,11 @@ function arrayify(x::CoDual{A}) where {A<:AbstractArray{<:BlasRealFloat}}
     return arrayify(primal(x), tangent(x))::Tuple{A,A}
 end
 arrayify(x::Array{P}, dx::Array{P}) where {P<:BlasRealFloat} = (x, dx)
+
+function arrayify(x::CoDual{A}) where {A<:AbstractArray{<:Union{ComplexF32, ComplexF64}}}
+    return arrayify(primal(x), tangent(x))
+end
```

Comment:
```suggestion
```
I think it should be fine to just change the element types permitted in the existing method of `arrayify` which accepts `CoDual`s from `BlasRealFloat` to `BlasFloat`.